libs/libwebsockets/Makefile

```diff
@@ -43,32 +43,37 @@ define Package/$(PKG_NAME)/Default
 	SECTION:=libs
 	CATEGORY:=Libraries
 	TITLE:=libwebsockets
-	DEPENDS:=+zlib
 	URL:=https://libwebsockets.org
 	MAINTAINER:=Karl Palsson <karlp@etactica.com>
```

Comment:
Please consider moving this to a global PKG_MAINTAINER var to cover the newly introduced minimal-server package.